platform/viewer/public/config/default.js

```diff
@@ -115,6 +115,11 @@ window.config = {
       label: 'W/L Preset 9',
       keys: ['9'],
     },
+    {
+      commandName: 'downloadAndZipSeriesOnActiveViewport',
```

Comment:
We do not want this enabled by default, you can take this out of the config.

Reply:
OK. I'll remove it in the following PR update.